databricks/koalas/tests/test_indexes.py

```diff
@@ -524,3 +524,66 @@ def test_argmax(self):
                 TypeError,
                 "reduction operation 'argmax' not allowed for this dtype"):
             kidx.argmax()
+
+    def test_monotonic(self):
+        # test monotonic_increasing & monotonic_decreasing for MultiIndex
+        datas = []
+
+        # increasing / decreasing ordered each index level with string
+        datas.append([('w', 'a'), ('x', 'b'), ('y', 'c'), ('z', 'd')])
+        datas.append([('w', 'd'), ('x', 'c'), ('y', 'b'), ('z', 'a')])
```

Comment:
What if

- `('x', 'd'), ('y', 'c'), ('y', 'b'), ('z', 'a')]`
- `('x', 'd'), ('y', 'b'), ('y', 'c'), ('z', 'a')]`

?

Reply:
i'll add them to test cases. thanks!

Reply:
hmm.. i think we should consider grouped previous level of index for each level of index >= 2.